apps/full-site-editing/full-site-editing-plugin/site-editor-experiment/index.php

```diff
@@ -127,6 +127,15 @@ function fse_plugin_edit_site_init( $hook ) {
 	apply_filters( 'template_include', null );
 	$settings['templateId'] = $_wp_current_template_id;
 
+	// @DOTCOM_CUSTOMIZATION - set up apiFetch properly for dotcom.
+	// NOT currently calling the script we want.
+	wp_enqueue_script(
+		'gutenberg-wpcom-apifetch',
+		'/wp-content/plugins/gutenberg-wpcom/gutenberg-wpcom-apifetch.js',
+		array( 'gutenberg-wpcom-script', 'jquery.wpcom-proxy-request', 'lodash', 'wp-api-fetch', 'wp-polyfill', 'wp-url' ),
+		time()
+	);
```

Comment:
We are thinking we need to enqueue this script to properly set up apiFetch for dotcom and get around that CORS error.

However, this required script isn't actually running with this enqueue set-up. 🤔